packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -55,11 +55,12 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
 
         if (label && labelPlacement && (labelPlacements.includes(labelPlacement))) {
             return html`
-                <span
+                <label
```

Comment:
It looks to me like the structure of the label(s) will be like this:
```html
<label for="toggle-switch">Leading label</label>
<label>
    <input id="toggle-switch" />
</label>
<label for="toggle-switch">Trailing label</label>
```

Is that right? I know the leading and trailing labels won't be showing at the same time but I'm not sure we need the label wrapping the input too.

Reply:
Will take a look to see if removing the wrapping works ok tomorrow 👍

Reply:
I think i will leave this as is for now. Snacks follow a similar structure although they use a span for for the actual label where text appears.